### DISCUSSION TITLE
Love the tool, though a suggestion

### DISCUSSION THREAD

**Discussion by JoppeMin** [sienori/simple-translate#305] [Ideas]
when using the Simple Translate's select-popup function the original language is not known, it would be a nice addition to see what language it was originally translated from, just faded in the corner of the popup.

![afbeelding](https://user-images.githubusercontent.com/18066600/119258111-9b66f000-bbc8-11eb-992e-9b9c2f920276.png)

**Comment by chwdWHO** [comment]
+1
